docs/app-portal.mdx

```diff
@@ -76,19 +78,23 @@ curl -X POST "https://api.svix.com/api/v1/auth/dashboard_access/app_Xzx8bQeOB1D1
 </TabItem>
 </CodeTabs>
 
-
 ## Embedding in your own dashboard
 
 The returned URL from the previous section can also be embedded in your own dashboard using an iframe.
 
-<img alt="Embedded Management UI screenshot" src={useBaseUrl("img/embedded-management-ui.png")} />
+<img
+  alt="Embedded App Portal screenshot"
+  src={useBaseUrl("img/embedded-management-ui.png")}
+/>
```

Comment:
I think we can just use normal markdown images now (docusaurus fixed it). No need to edit it, just pointed it out so you know for next time.